config/models.go

```diff
@@ -15,40 +15,41 @@ const (
-	LDKLogLevel              string `envconfig:"LDK_LOG_LEVEL" default:"3"`
-	LDKVssUrl                string `envconfig:"LDK_VSS_URL" default:"https://vss.getalbypro.com/vss"`
-	LDKListeningAddresses    string `envconfig:"LDK_LISTENING_ADDRESSES" default:"0.0.0.0:9735,[::]:9735"`
-	LDKTransientNetworkGraph bool   `envconfig:"LDK_TRANSIENT_NETWORK_GRAPH" default:"false"`
-	MempoolApi               string `envconfig:"MEMPOOL_API" default:"https://mempool.space/api"`
-	AlbyClientId             string `envconfig:"ALBY_OAUTH_CLIENT_ID" default:"J2PbXS1yOf"`
-	AlbyClientSecret         string `envconfig:"ALBY_OAUTH_CLIENT_SECRET" default:"rABK2n16IWjLTZ9M1uKU"`
-	BaseUrl                  string `envconfig:"BASE_URL"`
-	FrontendUrl              string `envconfig:"FRONTEND_URL"`
-	LogEvents                bool   `envconfig:"LOG_EVENTS" default:"true"`
-	AutoLinkAlbyAccount      bool   `envconfig:"AUTO_LINK_ALBY_ACCOUNT" default:"true"`
-	PhoenixdAddress          string `envconfig:"PHOENIXD_ADDRESS"`
-	PhoenixdAuthorization    string `envconfig:"PHOENIXD_AUTHORIZATION"`
-	GoProfilerAddr           string `envconfig:"GO_PROFILER_ADDR"`
-	DdProfilerEnabled        bool   `envconfig:"DD_PROFILER_ENABLED" default:"false"`
-	EnableAdvancedSetup      bool   `envconfig:"ENABLE_ADVANCED_SETUP" default:"true"`
-	AutoUnlockPassword       string `envconfig:"AUTO_UNLOCK_PASSWORD"`
-	LogDBQueries             bool   `envconfig:"LOG_DB_QUERIES" default:"false"`
-	BoltzApi                 string `envconfig:"BOLTZ_API" default:"https://api.boltz.exchange"`
+	Relay                              string `envconfig:"RELAY" default:"wss://relay.getalby.com/v1"`
+	LNBackendType                      string `envconfig:"LN_BACKEND_TYPE"`
+	LNDAddress                         string `envconfig:"LND_ADDRESS"`
+	LNDCertFile                        string `envconfig:"LND_CERT_FILE"`
+	LNDMacaroonFile                    string `envconfig:"LND_MACAROON_FILE"`
+	Workdir                            string `envconfig:"WORK_DIR"`
+	Port                               string `envconfig:"PORT" default:"8080"`
+	DatabaseUri                        string `envconfig:"DATABASE_URI" default:"nwc.db"`
+	JWTSecret                          string `envconfig:"JWT_SECRET"`
+	LogLevel                           string `envconfig:"LOG_LEVEL" default:"4"`
+	LogToFile                          bool   `envconfig:"LOG_TO_FILE" default:"true"`
+	Network                            string `envconfig:"NETWORK"`
+	LDKNetwork                         string `envconfig:"LDK_NETWORK"`
+	LDKEsploraServer                   string `envconfig:"LDK_ESPLORA_SERVER" default:"https://electrs.getalbypro.com"` // TODO: remove LDK prefix
+	LDKGossipSource                    string `envconfig:"LDK_GOSSIP_SOURCE"`
+	LDKLogLevel                        string `envconfig:"LDK_LOG_LEVEL" default:"3"`
+	LDKMaxChannelSaturationPowerOfHalf string `envconfig:"LDK_MAX_CHANNEL_SATURATION" default:"2"`
```

Comment:
can this be the proper type instead of a string?

Reply:
Should I also change this for other fields? For example LDKLogLevel?